pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -386,3 +386,114 @@ func createEcDoubleSlopeV1Hinter(resolver hintReferenceResolver) (hinter.Hinter,
 
 	return newEcDoubleSlopeV1Hint(point), nil
 }
+
+func newEcDoubleAssignNewXV1(slope, point hinter.ResOperander) hinter.Hinter {
```

Comment:
Rename `newEcDoubleAssignNewXV1` to `newEcDoubleAssignNewXV1Hint` to match existing naming conventions.

Reply:
Done! @har777